client/apps/articles_list/client/client.coffee

```diff
@@ -0,0 +1,94 @@
+Backbone = require 'backbone'
+_ = require 'underscore'
+Q = require 'bluebird-q'
+React = require 'react'
+jqueryOnInfitireScroll = require('jquery-on-infinite-scroll')
+{ div, nav, a, h1 } = React.DOM
+Article = require '../../../models/article.coffee'
+FilterSearch = require '../../../components/filter_search/index.coffee'
+query = require '../query.coffee'
+sd = require('sharify').data
+request = require 'superagent'
+icons = -> require('../icons.jade') arguments...
+
+module.exports.ArticlesListView = ArticlesListView = React.createClass
+  getInitialState: ->
+    articles: @props.articles or []
+    published: @props.published or true
+    offset: 0
+
+  componentDidMount: ->
+    canLoadMore = _.debounce @canLoadMore, 300
+    $.onInfiniteScroll canLoadMore
+
+  canLoadMore: ->
+    return unless !$('.filter-search__input').val()
```

Comment:
in coffeescript, you should use `not` instead of `!` 

but either way i think this can be `return if $('.filter-search__input').val()`?